tess-mediator.js

```diff
@@ -41,7 +42,7 @@ function writeDb(db) {
 function checkDb() {
     const db = loadDb();
 
-    if (db.height > 0 && (db.height - db.scanned) < FIRST) {
+    if (!db.first || db.first < FIRST) {
```

Comment:
This fixes an unrelated bug where it was deleting the mediator db on every startup.